openmc/deplete/results.py

```diff
@@ -157,6 +158,59 @@ def get_atoms(
 
         return times, concentrations
 
+    def get_mass(self,
+        mat: typing.Union[Material, str],
+        nuc: str,
+        mass_units: str = "g",
+        time_units: str = "s"
+    ) -> Tuple[np.ndarray, np.ndarray]:
+        """Get mass of nuclides over time from a single material
+
+        .. versionadded:: 0.13.4
+
+        Parameters
+        ----------
+        mat : openmc.Material, str
+            Material object or material id to evaluate
+        nuc : str
+            Nuclide name to evaluate
+        mass_units : {"g", "g/cm3", "kg"}, optional
+            Units for the returned mass.
+        time_units : {"s", "min", "h", "d", "a"}, optional
+            Units for the returned time array. Default is ``"s"`` to
+            return the value in seconds. Other options are minutes ``"min"``,
+            hours ``"h"``, days ``"d"``, and Julian years ``"a"``.
+
+        Returns
+        -------
+        times : numpy.ndarray
+            Array of times in units of ``time_units``
+        mass : numpy.ndarray
+            Mass of specified nuclide in units of ``mass_units``
+
+        """
+        cv.check_value("mass_units", mass_units, {"g", "g/cm3", "kg"})
```

Comment:
```suggestion
        cv.check_value("mass_units", mass_units, {"g", "g/cm3", "kg", "kg/m3"})
```
If we do want more units, then this line might be changed as wll